osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json

```diff
@@ -989,6 +989,90 @@
           "access": "R"
         }
       ]
+    },
+    {
+      "tag": "ACTIVITY_CALENDAR",
+      "description": "Activity calendar",
+      "note": null,
+      "class-id": 20,
+      "version": 0,
+      "obis": "0.0.13.0.0.255",
+      "group": "ABSTRACT",
+      "meterTypes": ["SP","PP"],
+      "attributes": [
+        {
+          "id": 2,
+          "description": "calendar_name_active",
+          "datatype": "octet-string",
+          "valuetype": "DYNAMIC",
+          "value": null,
+          "access": "R"
+        },
+        {
+          "id": 3,
+          "description": "season_profile_active",
+          "datatype": "array",
+          "valuetype": "DYNAMIC",
+          "value": null,
+          "access": "R"
+        },
+        {
+          "id": 4,
+          "description": "week_profile_table_active",
+          "datatype": "array",
+          "valuetype": "DYNAMIC",
+          "value": null,
+          "access": "R"
+        },
+        {
+          "id": 5,
+          "description": "day_profile_table_active",
+          "datatype": "array",
+          "valuetype": "DYNAMIC",
+          "value": null,
+          "access": "R"
+        },
+        {
+          "id": 6,
+          "description": "calendar_name_passive",
+          "datatype": "octet-string",
+          "valuetype": "DYNAMIC",
```

Comment:
The "passive" attributes are not updated by the meter itself. The type should be "SET_BY_CLIENT".